python/setup.py

```diff
@@ -18,7 +18,7 @@
 jupyter = ["matplotlib", "jupyterlab"]
 aws = ["boto"]
 docs = ["sphinx"]
-youtube = ["pafy", "youtube_dl"]
+youtube = ["pafy", "youtube_dl", "ffmpeg-python"]
```

Comment:
this shouldn't go in `youtube` extras. 
how big is the ffmpeg package?
@eddyxu should we think about ffmpeg as an essential library like say opencv? or an optional library?